data-processing/tests/test_extract_definitions.py

```diff
@@ -5,15 +5,15 @@
 import pytest
 from entities.definitions.commands.detect_definitions import (
     DefinitionDetectionModel,
-    consolidate_term_definitions,
+    consolidate_keyword_definitions,
     get_abbreviations,
     get_symbol_nickname_pairs,
 )
 
 
 @pytest.fixture(scope="module")
 def model():
-    model = DefinitionDetectionModel()
+    model = DefinitionDetectionModel(['TERM-DEF', 'ABBR-EXP', 'SYM-NICK'])
```

Comment:
could these args just be hard-coded into the DefinitionDetectionModel? Is there a circumstance under which we would want to change the task names?